src/commands/domains/transfer-in.ts

```diff
@@ -56,11 +59,39 @@ export default async function transferIn(
     return 1;
   }
 
+  const availableStamp = stamp();
+  const [domainPrice, { transferrable }] = await Promise.all([
+    getDomainPrice(client, domainName),
+    checkTransfer(client, domainName)
+  ]);
+  if (domainPrice instanceof ERRORS.UnsupportedTLD) {
+    output.error(`The TLD for ${param(domainName)} is not supported.`);
+    return 1;
+  }
+  if (!transferrable) {
+    output.error(`The domain ${param(domainName)} is not transferable.`);
+    return 1;
+  }
+
+  const { price } = domainPrice;
+  output.log(
+    `The domain ${param(domainName)} is ${chalk.underline(
+      'available'
```

Comment:
Not sure if `available` is the best way to say this

Reply:
cc @timothyis